pkg/controllers/controllers.go

```diff
@@ -60,7 +61,9 @@ func NewControllers(
 ) []controller.Controller {
 
 	cluster := state.NewCluster(clock, kubeClient)
-	p := provisioning.NewProvisioner(kubeClient, recorder, cloudProvider, cluster)
+	SIHeap := orb.NewSchedulingInputHeap()
+	SMHeap := orb.NewSchedulingMetadataHeap()
+	p := provisioning.NewProvisioner(kubeClient, recorder, cloudProvider, cluster, SIHeap, SMHeap)
```

Comment:
Wdyt about using a generic `orb.Inject()`

Reply:
Or, I think what happens is here is that orb is instantiated, we can then call `Inject()` for a `NewProvisioner` and then pass a new orb controller to `[]controller.Controller{`. See the `metricsnode` controller with its `cluster` param.